src/Zns.ts

```diff
@@ -38,7 +36,6 @@ const UrlMap = {
 
 const UrlNetworkMap = (url: string) => invert(UrlMap)[url];
 
-/** @internal */
 export default class Zns extends NamingService {
```

Comment:
We should keep all of that internal.

Reply:
With the whitelisting of files to typedoc we don't really need to specify something is internal anymore. The Zns or any other naming service classes are not shown up in the docs